functions/Get-DbaRunningJob.ps1

```diff
@@ -4,18 +4,16 @@ function Get-DbaRunningJob {
 			Returns all non-idle Agent jobs running on the server.
 
 		.DESCRIPTION
-			This function returns agent jobs that active on the SQL Server instance when calling the command. The information is gathered the SMO JobServer.jobs and be returned either in detailed or standard format.
+			This function returns agent jobs that active on the SQL Server instance when calling the command. The information is gathered
+			from the SMO JobServer.jobs and can be returned either in detailed or standard format.
```

Comment:
@josh-simar we are not adding line breaks to the documentation/help.

Reply:
Agree with @wsmelton. Soft wrapping should be handled by your editor and the console when the CBH is rendered to the user.  I've been removing manual line breaks as part of my CBH cleanup.